src/eval-frame/components/eval-container.jsx

```diff
@@ -14,10 +15,24 @@ export class EvalContainerUnconnected extends React.Component {
   static propTypes = {
     reportOnly: PropTypes.bool.isRequired,
   }
+  state = {
+    hasError: false,
+    error: null,
+    errorInfo: null,
+  }
+  componentDidCatch(error, info) {
+    this.setState({
+      hasError: true,
+      error,
+      errorInfo: info,
+    })
+    console.log(error, info)
+  }
+
 
   render() {
     const { reportOnly } = this.props
-    return (
+    let render = (
```

Comment:
I'm not sure why you're putting this content in a temporary and returning a fragment?

I think you could just put the `if (this.state.hasError)` at the top and just return the error if that's what's going on.

Reply:
Got it I do the same as you said.

Reply:
again, please don't resolve until you've landed a new commit which fixes the issue